sas9.4-viya/azuredeploy.json

```diff
@@ -0,0 +1,3683 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "SAS Application Name": {

```

Comment:
remove spaces and use camel casing for all parameter/variable/output names

Reply:
Removed spaces in between for all possible parameters variables and output names.